changelog.md

```diff
@@ -3,20 +3,28 @@
 ## Breaking changes
 
 - Do not automatically derive size and caption for `from_neo4j` and `from_gql_create`. Use the `size_property` and `node_caption` parameters to explicitly configure them.
+- Change API of integrations to only provide basic parameters. Any further configuration should happen ons the Visualization Graph object:
+  - `from_gds`
+    - Drop parameters size_property, node_radius_min_max. `Use VG.resize_nodes(property=...)` instead
+    - rename additional_node_properties to node_properties
+    - Dont derive fields from properties. Use `VG.map_properties_to_fields` instead
+  - `from_pandas`
+    - Drop `node_radius_min_max` parameter. `VG.resize_nodes(...)` instead
 
 ## New features
 
-- Allow to include db node properties in addition to the properties in the GDS Graph. Specify `additional_db_node_properties` in `from_gds`.
-
+- Allow to include db node properties in addition to the properties in the GDS Graph. Specify `db_node_properties` in `from_gds`.
 
 ## Bug fixes
 
 - fixed a bug in `from_neo4j`, where the node size would always be set to the `size` property.
 - fixed a bug in `from_neo4j`, where the node caption would always be set to the `caption` property.
+- Color nodes in `from_snowflake` only if there are less than 13 node tables used. This avoids reuse of colors for different tables.
```

Comment:
This doesn't quite seem like a bugfix? Just a change? Should we have the same color behaviour for the other constructors then too?

Reply:
yes i would suggest to have the same coloring behavior in neo4j and gql. i added it to gds in this PR.

Will do on a follow up PR